tfjs-backend-webgpu/src/conv2d_mm_webgpu.ts

```diff
@@ -38,17 +38,16 @@ export class Conv2DMMProgram implements WebGPUProgram {
   hasPreluActivationWeights: boolean;
   fitA: boolean;
   fitB: boolean;
+  isChannelsLast: boolean;
 
   constructor(
       convInfo: backend_util.Conv2DInfo, addBias = false,
       activation: backend_util.Activation = null,
       hasPreluActivationWeights = false) {
     this.outputShape = convInfo.outShape;
-
-    util.assert(
-        convInfo.dataFormat === 'channelsLast',
-        () => 'TODO: NCHW is unimplemented');
-    this.dispatchLayout = {x: [3], y: [1, 2], z: [0]};
+    this.isChannelsLast = convInfo.dataFormat === 'channelsLast';
+    this.dispatchLayout = this.isChannelsLast ? {x: [3], y: [1, 2], z: [0]} :
+                                                {x: [1], y: [2, 3], z: [0]};
```

Comment:
If we simply swaps the dispatch layout, does it mean that global memory reading and writing are not coalesced access for `channelsFirst`?
Is it possible that we keep the layout as {x: [2, 3], y: [1], z: [0]} to get a good perf?